### ISSUE TITLE
username validation

### ISSUE THREAD

**Issue by faust64** [PeerioTechnologies/peerio-client#317]
As spotted today, a new user wasn't able to register his new account.
The problem was in his username, which included a space character.
The client should raise an error here, though it doesn't.

**Comment by anri-asaturov** [comment]
valudateUsername treats 'hello world' as an attempt to validate 2 usernames 'hello' and 'world'

**Comment by faust64** [comment]
I guess it could be related to the way the message is sent from the client.
Checking from my unit tests:

```
INFO APIcall
 Methond: validateUsername
 Data:  {"username":"eric cartmann"}
 Resp:  {"error":406,"message":"username fails to match the required pattern"}
Unhandled rejection Error: Failed to close test account #1 [{"error":406,"message":"username fails to match the required pattern"}]
    at /root/git/peerio-server/test/02-username.test.js:45:12
    at tryCatcher (/root/git/peerio-server/node_modules/bluebird/js/release/util.js:11:23)
...
```

**Comment by faust64** [comment]
Actually, forcing api v1, sending the same message, I have a single empty object coming back (?!)

**Comment by faust64** [comment]
Back on my v1 test: my bad, was actually testing against v2.
Fixed by Floh, on the server (see 3a6e73dafcaac16c900465d0916d413155a489b7).
